.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
Kindy add node_modules also to this file as per the requirements of the project

Reply:
Hello thanks for the review. I did add the test.md file but it is ignored from the repo as instructed